lib/logux.rb

```diff
@@ -51,11 +51,12 @@ def initialize(msg, meta: nil)
   configurable :logux_host, :verify_authorized,
                :password, :logger,
                :on_error, :auth_rule,
-               :render_backtrace_on_error
+               :render_backtrace_on_error, :logux_logger
```

Comment:
let name it just `logger`? `Logux.logux_logger` sounds overloaded

Reply:
But we already have logger: https://github.com/logux/logux_rails/blob/master/lib/logux.rb#L59-L62

Reply:
👍

Reply:
Adding one more logger looks confusedly. Why don't we just use the logger. If it is not set - instantiate our own?

Reply:
@dsalahutdinov ok 👍